tests/constbuffer_ut/constbuffer_ut.c

```diff
@@ -2562,4 +2562,144 @@ TEST_FUNCTION(CONSTBUFFER_from_buffer_with_2_size_unhappy_path)
     ///clean
 }
 
+/*Tests_SRS_CONSTBUFFER_51_001: [ If `size` is 0, then CONSTBUFFER_create_writiable_handle shall fail and return NULL. ]*/
+TEST_FUNCTION(CONSTBUFFER_create_writable_handle_invalid_args_fails)

```

Comment:
Consider adding a /* CONSTBUFFER_CreateWritableHandle */ comment which can act as a section delimiter for readability. /consider

Reply:
Added